alf/test/case.py

```diff
@@ -53,6 +53,6 @@ def assertTensorNotClose(self, t1, t2, epsilon=1e-6, msg=None):
                               'First argument is not a Tensor')
         self.assertIsInstance(t2, torch.Tensor,
                               'Second argument is not a Tensor')
-        if not (torch.min(torch.abs(t1 - t2)) > epsilon):
+        if not (torch.max(torch.abs(t1 - t2)) > epsilon):
```

Comment:
There seems to be a bug for checking the `closeness` of two tensors.

Reply:
Suggest directly checking `if torch.max(torch.abs(t1 - t2)) < epsilon` for readability.